src/fakedht/request/GetNumConnections.d

```diff
@@ -50,7 +51,8 @@ public scope class GetNumConnections : Protocol.GetNumConnections
 
     override protected NumConnectionsData getConnectionsData ( )
     {
-        assert (false,
+        enforce(false,
             "GetNumConnections is not supported by the fake DHT node");
+        return NumConnectionsData.init;
```

Comment:
I'll allow this to be merged in this commit ;-)

Reply:
:roll_eyes: